examples/input/gamepad_input_events.rs

```diff
@@ -15,25 +16,31 @@ fn main() {
 }
 
 fn gamepad_events(
-    mut gamepad_connection_events: EventReader<GamepadConnectionEvent>,
-    mut gamepad_axis_events: EventReader<GamepadAxisChangedEvent>,
-    mut gamepad_button_events: EventReader<GamepadButtonChangedEvent>,
+    mut connection_events: EventReader<GamepadConnectionEvent>,
+    mut axis_changed_events: EventReader<GamepadAxisChangedEvent>,
+    mut button_changed_events: EventReader<GamepadButtonChangedEvent>,
+    mut button_input_events: EventReader<GamepadButtonInput>,
```

Comment:
I think this would benefit from a comment explaining the difference between `GamepadButtonInput` and `GamepadButtonChangedEvent`